SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2377,15 +2371,16 @@ void USpatialNetDriver::HandleStartupOpQueueing(const TArray<Worker_OpList*>& In
 		bIsReadyToStart = FindAndDispatchStartupOpsClient(InOpLists);
 	}
 
+	QueuedStartupOpLists.Append(MoveTemp(InOpLists));
```

Comment:
Any reason for moving this?

Reply:
Because you need to transfer ownership after reading from it.

Reply:
Previously this wasn't a problem because the transfer wasn't destructive, but now it requires moving the list so it can't go above any more.